plugins/polio/api/campaigns/campaigns.py

```diff
@@ -134,7 +133,9 @@ def get_top_level_org_unit_id(self, campaign):
 
     def get_general_status(self, campaign):
         now_utc = timezone.now().date()
-        for round in campaign.rounds.all().order_by("-number"):
+        ordered_rounds = list(campaign.rounds.all())
+        ordered_rounds.sort(key=lambda x: x.number, reverse=True)
```

Comment:
I would never have thought it was cheaper to do the ordering this way

Reply:
It prevents additional requests to the database server while we already have the data. 

Going through the network is usally an order of magnitude slower than accessing memory. 

Here it works because I prefetched all the rounds in prefetch_related. If that was not there, we would do one request per campaign